app/models/dataset_record/darwin_core/occurrence.rb

```diff
@@ -731,9 +744,19 @@ def parse_identification_class
     scientific_name = get_field_value(:scientificName)&.gsub(/\s+/, ' ')
     type_scientific_name = type_status&.[](:scientificName)&.gsub(/\s+/, ' ')
 
-    type_material = {
-      type_type: type_status[:type].downcase
-    } if scientific_name && type_scientific_name&.delete_prefix!(scientific_name)&.match(/^\W*$/)
+
+    if scientific_name && type_scientific_name.present?
+      if type_scientific_name&.delete_prefix!(scientific_name)&.match(/^\W*$/)
+        type_material = {
+          type_type: type_status[:type].downcase
```

Comment:
Feels like you might compare (maybe done elsewhere) that the value here is a legal `type_type` (`TypeMateria::ICZN_TYPES` for example)?

Reply:
`type_type` is validated during create, which is called here: https://github.com/SpeciesFileGroup/taxonworks/blob/57894801c017e782f8a8ebf6ac4cc851cc40d292/app/models/dataset_record/darwin_core/occurrence.rb#L134-L140

Type material was originally designed to be skipped if it failed. If that's no longer the intent we can make an issue for it.